README.md

```diff
@@ -36,215 +51,26 @@ Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs
 
 * Other OS
 
-  See the installation instructions for the [latest release](https://github.com/jimmidyson/minishift/releases).
+  [TODO - CANNOT FIND INSTRUCTIONS] See the [installation instructions for the latest release](https://github.com/minishift/minishift/releases).
 
 ## Quickstart
 
```

Comment:
I would not extract the quickstart. This is part of the essential info. This should be right on the top after the installation. I agree it is a big long for now, so I suggest to shorten it and boil it down to the essentials. Start minishift and deploy app. The waiting for pods and the curl command are not really required and we can condense this. 

Also, I don't think:

     oc run hello-minishift --image=gcr.io/google_containers/echoserver:1.4 --port=8080 --expose --service-overrides='{"apiVersion": "v1", "spec": {"type": "NodePort"}}'

actually works. We should update the example. See also https://github.com/minishift/minishift/pull/103